VideoLocker/src/main/java/org/edx/mobile/module/analytics/ISegmentImpl.java

```diff
@@ -128,7 +128,7 @@ public Properties trackVideoLoading(String videoId, String courseId, String unit
     public Properties trackVideoPlaying(String videoId, Double currentTime,
             String courseId, String unitUrl){
         try{
```

Comment:
It would be great if all these generic catch blocks would be removed from here and the tracker as well…